### PR TITLE
chore: Bump wasmer to 1.0.0-beta1

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: packages/api/src/wasi.rs [hunk 1]
```diff
@@ -447,53 +447,6 @@ impl Environment {
...
-
-                Ok(())
-            }
-
-            _ => Err(to_py_err::<PyTypeError, _>(
-                "Can only set a `Memory` object to `Environment.memory`",
-            )),
-        }
-    }
-
```
Comment (Hywan) [inline packages/api/src/wasi.rs:496]: I think you'll need to update tests too.

Comment (jubianchi) [reply packages/api/src/wasi.rs:496]: done :)

**Review by MarkMcCaskey — APPROVED** [review]
Looks good on my end!

For Python API-specific review you might want to get additional review if you're unsure about any changes you made!

**Comment by Hywan** [comment]
Tips: You can take previous release notes from https://github.com/wasmerio/wasmer-python/releases directly.

**Pull request merged**